CHANGELOG.md

```diff
@@ -13,7 +13,6 @@ Possible log types:
 - `[docs]` for documentation changes.
 - `[chore]` for maintenance work.
 
-
```

Comment:
Could you please revert this reformatting of an otherwise unchanged file?

Reply:
Done